### PR TITLE
Fixed: No information in logs about category when its added to CF

### PR DESCRIPTION
**Pull request by mingkaic**
Change to make logs more informative:
Format from 
```
DEBU[0006]` Policy update                                 update=policy update: 93012340-27cc-47af-97a6-59cd654cf2cc - 7b9096d1-3ad1-4fbf-940d-193d4df780e1
commands:
- Add ("Default"/"query"/"DFPRNetworks"/"0000002e"/"Default")
- Add ("Default"/"query"/"DFPRNetworks"/"CIDRNetworks"/"0000002e"/"Default")
```
to
```
DEBU[0006] Policy update                                 update=policy update: 94db536a-f402-4ea9-a9cb-a3e3445bddf9 - 3664a605-97d3-427d-ac21-4acf8de56f96
commands:
- Add policy "Default" rules([Default News Blogs Tobacco Drugs]) to
  path: ("Default"/"query"/"DFPRNetworks"/"00000001"/"Default")
- Add policy "Default" rules([Default News Blogs Tobacco Drugs]) to
  path: ("Default"/"query"/"DFPRNetworks"/"CIDRNetworks"/"00000001"/"Default")
```

### CODE REVIEW

**Comment by mingkaic** [comment]
@UladzimirTrehubenka Can you review this please? I can't add reviewers for this pull request.

File: pdp/policy.go [hunk 1]
```diff
@@ -59,8 +59,16 @@ func NewPolicy(ID string, hidden bool, target Target, rules []*Rule, makeRCA Rul
...
 
 func (p *Policy) describe() string {
+	ruleIDs := make([]string, len(p.rules))
+	ruleIdx := 0
+	for _, rule := range p.rules {
+		if ruleID, ok := rule.GetID(); ok {
+			ruleIDs[ruleIdx] = ruleID
+			ruleIdx++
+		}
+	}
```
Comment (mingkaic) [inline pdp/policy.go]: Is it worthwhile adding a limit to the number of rules to display? Can policies have a lot of rules?

**Comment by mingkaic** [comment]
https://jira.inca.infoblox.com/browse/JANUS-12954

**Comment by UladzimirTrehubenka** [comment]
Don't use any private internals in open source repo.

File: pdp/policy.go (hunk 1 above)
Comment (vasili-v) [reply pdp/policy.go]: Yes, they can. And yes you have to.

File: pdp/policy.go [hunk 2]
```diff
@@ -59,8 +59,16 @@ func NewPolicy(ID string, hidden bool, target Target, rules []*Rule, makeRCA Rul
...
 	if pid, ok := p.GetID(); ok {
-		return fmt.Sprintf("policy %q", pid)
+		return fmt.Sprintf("policy %q rules(%v)", pid, ruleIDs[:ruleIdx])
```
Comment (vasili-v) [inline pdp/policy.go]: The way you format message is neither human nor machine readable. As the list is anyway limited please make it like "x", "y", ..., "z".

**Pull request merged**